app/views/index.slim

```diff
@@ -20,7 +20,7 @@ form method="POST" action="/"
 		h1 = "On #{provider_name}"
 
 		ul
-			- provider.repositories.each do |repo|
+			- provider.repositories(repos).each do |repo|
```

Comment:
What do you think about making this a keyword argument so it is more clear when we call it?

`provider.repositories(exclude: repos)` seems more self explanatory to me